openshift-environment-driver/src/main/java/org/jboss/pnc/environment/openshift/OpenshiftStartedEnvironment.java

```diff
@@ -85,6 +87,9 @@
     private Route route;
     private Service sshService;
 
+    @Inject
+    private Configuration configuration;
```

Comment:
OpenshiftStartedEnvironment is also instantited using new, thus not managed by the CDI.